test/TextToBinary.cpp

```diff
@@ -62,6 +62,27 @@ TEST(GetWord, Simple) {
   EXPECT_EQ("abc", AssemblyContext(AutoText("abc\n"), nullptr).getWord());
 }
 
+TEST(GetWord, MultipleWords) {
+  const std::string contents = "abc\n\rdef\t\vghi   l12";
+  spv_text_t text = {contents.data(), contents.size()};
+  auto context = AssemblyContext(&text, nullptr);
+
+  EXPECT_EQ("abc", context.getWord());
+
+  spv_position_t position = {};
+  position = {1, 1, contents.find('d')};
+  context.setPosition(position);
+  EXPECT_EQ("def", context.getWord()) << position.index;
+
+  position = {1, 5, contents.find('g')};
```

Comment:
Changed. Previously because of line endings, `index` is not the same as `column`.